lib/web_ui/dev/environment.dart

```diff
@@ -150,8 +150,8 @@ class Environment {
   ///
   /// This is where compiled output goes.
   io.Directory get webUiBuildDir => io.Directory(pathlib.join(
-        webUiRootDir.path,
-        'build',
+        outDir.path,
```

Comment:
Can we copy to both locations as a transitional step to not break the current tests?

Reply:
I don't think we'll have to do that. AFAIK the current tests will not be broken by this.